core/storage/trie/impl/polkadot_node.hpp

```diff
@@ -81,6 +68,20 @@ namespace kagome::storage::trie {
     std::array<std::shared_ptr<PolkadotNode>, kMaxChildren> children;
   };
 
+
+  struct LeafNode : public PolkadotNode {
+    LeafNode() = default;
+    LeafNode(common::Buffer key_nibbles, common::Buffer value)
```

Comment:
probably need to add inline

Reply:
I think the compiler will do fine on its own.